src/backends/custom/custom.h

```diff
@@ -249,6 +287,15 @@ TRTIS_CUSTOM_EXPORT int CustomExecute(
     void* custom_context, uint32_t payload_cnt, CustomPayload* payloads,
```

Comment:
In the description of CustomExecute.
"Execute the custom model using the version 1 implementation of the execute interface. This function must be defined when the custom backend returns 1 from CustomVersion (or when CustomVersion is not defined). ..."